src/Templates/Microsoft.CodeAnalysis.AnalyzerTemplate/MyProject.Analyzer.CodeFixes/MyProject.Analyzer.CodeFixes.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <IsPackable>false</IsPackable>
+    <RootNamespace>MyProject</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="$(MicrosoftCodeAnalysisCSharpVersionVS2019)" />
```

Comment:
For (all?) dotnet repos it comes from eng/versions.props, but good question if it will be replaced when packaged. I haven't packaged them yet so will test and make sure. I wanted to find the best way for us to use the right version with the SDK